src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values()[index].name();
```

Comment:
1) There is no need to return a string, instead return an enum instance here.
2) Also consider extracting new Random to the field on the class level, so you can reuse it and not create every time you call this method.
3) "Colors.values()" used twice here, consider extracting it to the field too